sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample1_RecognizeReceipts.md

```diff
@@ -0,0 +1,74 @@
+# Recognize receipts
+
+This sample demonstrates how to recognize and extract common fields from US receipts, using a pre-trained receipt model.
+
+To get started you'll need a Cognitive Services resource or a Form Recognizer resource.  See [README][README] for prerequisites and instructions.
+
+## Creating a `FormRecognizerClient`
+
+To create a new `FormRecognizerClient` you need the endpoint and credentials from your resource. In the sample below you'll use a Form Recognizer API key credential by creating an `AzureKeyCredential` object, that if needed, will allow you to update the API key without creating a new client.
+
+You can set `endpoint` and `apiKey` based on an environment variable, a configuration setting, or any way that works for your application.
+
+```C# Snippet:CreateFormRecognizerClient
+string endpoint = "<endpoint>";
+string apiKey = "<apiKey>";
+var credential = new AzureKeyCredential(apiKey);
+var client = new FormRecognizerClient(new Uri(endpoint), credential);
+```
+
+## Recognize receipts from a URI
+
+To recognize receipts from a URI, use the `StartRecognizeReceiptsFromUri` method. The returned value is a collection of `RecognizedReceipt` objects -- one for each page in the submitted document.
+
+```C# Snippet:FormRecognizerSample1RecognizeReceiptFileFromUri
+Response<IReadOnlyList<RecognizedReceipt>> receipts = await client.StartRecognizeReceiptsFromUri(new Uri(receiptUri)).WaitForCompletionAsync();
+foreach (var receipt in receipts.Value)
+{
+    USReceipt usReceipt = receipt.AsUSReceipt();
+
+    string merchantName = usReceipt.MerchantName;
+    DateTime transactionDate = usReceipt.TransactionDate;
+    IReadOnlyList<USReceiptItem> items = usReceipt.Items;
+    float subtotal = usReceipt.Subtotal;
+    float tax = usReceipt.Tax;
+    float tip = usReceipt.Tip;
+    float total = usReceipt.Total;
+
+    Console.WriteLine($"Recognized USReceipt fields:");
+    Console.WriteLine($"    Merchant Name: '{merchantName}', with confidence {usReceipt.MerchantName.Confidence}");
+    Console.WriteLine($"    Transaction Date: '{transactionDate}', with confidence {usReceipt.TransactionDate.Confidence}");
+
+    for (int i = 0; i < items.Count; i++)
+    {
+        USReceiptItem item = usReceipt.Items[i];
+        Console.WriteLine($"    Item {i}:  Name: '{item.Name.Value}', Quantity: '{item.Quantity?.Value}', TotalPrice: '{item.TotalPrice.Value}'");
```

Comment:
Can you include the individual item's price?

Reply:
added